packages/svelte/src/compiler/phases/3-transform/client/visitors/shared/fragment.js

```diff
@@ -83,7 +83,14 @@ export function process_children(nodes, initial, is_element, { visit, state }) {
 		} else if (has_state && !within_bound_contenteditable) {
 			state.update.push(update);
 		} else {
-			state.init.push(b.stmt(b.assignment('=', b.member(id, 'nodeValue'), value)));
+			// if the expression is inlinable we just push it to the template
+			if (is_inlinable_expression(sequence, state.scope) && !is_text) {
```

Comment:
it might be a bit more performant to do the simple check first and only call the function if needed
```suggestion
			if (!is_text && is_inlinable_expression(sequence, state.scope)) {
```

Reply:
Oh yeah